packages/api/lib/writeRecords/write-granules.js

```diff
@@ -518,6 +518,89 @@ const writeGranuleRecordAndPublishSns = async ({
   });
 };
 
+/**
+ * Update granule record status in DynamoDB, PostgreSQL, Elasticsearch.
+ * Publish SNS event for updated granule.
+ *
+ * @param {Object}  params
+ * @param {Object}  params.apiGranuleRecord      - API Granule object to write to
+ *                                                 the database
+ * @param {Object}  params.postgresGranule       - PostgreSQL granule record to write
+ *                                                 to database
```

Comment:
```suggestion
 * @param {Object}  params.postgresGranule       - PostgreSQL granule
```

This isn't what is written to PG. It's really only used for the `cumulus_id` propertly